L1Trigger/L1TGlobal/interface/AXOL1TLTemplate.h

```diff
@@ -52,9 +52,13 @@ class AXOL1TLTemplate : public GlobalCondition {
 public:
   inline const std::vector<ObjectParameter>* objectParameter() const { return &m_objectParameter; }
 
+  inline const std::string* modelVersion() const { return &m_modelVersion; }
```

Comment:
Why return a pointer instead of reference?
```suggestion
  inline const std::string& modelVersion() const { return m_modelVersion; }
```

Reply:
no particular reason: I was just following the format of the object parameter definition. I can commit your suggestion

Reply:
Thanks. Maybe my question is then more about the existing pattern.